PcsConfig/startcontainers.sh

```diff
@@ -0,0 +1,21 @@
+﻿#!/usr/bin/env bash
```

Comment:
There is already [containers.sh](https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet-tests/blob/master/scripts/containers.sh) script to which you can pass in `start` or `stop` value. We don't need these 2 files.

Reply:
done